README.md

```diff
@@ -1,18 +1,32 @@
 # CAPI2-BSP (CAPI version 2.0 board support)
-Create CAPI 2.0 board infrastructure including PSL as IP container for various FPGA-cards.
+## What is CAPI 2.0?
+The majority of computational accelerator systems use PCI Express (PCIe) to connect to the main processor
+units via the I/O subsystem. The IBM **C**oherent **A**ccelerator **P**rocessor **I**nterface (CAPI) provides an alternative
+that removes the complexity and overhead of the I/O subsystem to enable higher system performance.
 
-Depends on PSL sources that can be obtained as zip archive from `<TBD>`
+While the first version of CAPI was designed for acceleration in a POWER8 system, its successor CAPI 2.0 is the CAPI version for POWER9 systems.
+
+For more information on CAPI, see https://developer.ibm.com/linuxonpower/capi/
+
+## Board support and PSL
+The Power Service Layer (PSL) is the component for FPGAs that provides access to CAPI.
+
+This repository contains the necessary resources for creating a Xilinx Vivado IP container that integrates the PSL component into an FPGA card specific infrastructure for CAPI 2.0.
 
 Currently, support for the following cards is implemented:
 * [AD9V3](./AD9V3) (AlphaData 9V3)
 * [N250SP](./N250SP) (Nallatech 250S+)
 
-## Creating container via make process
-Before creating the CAPI board support IP container, the P9 PSL (PSL9) sources have to be made available for the build process.
-This can be done by copying the PSL9 IP core archive or the encrypted PSL9 source file archive to the subdirectory [psl](./psl)
-(see section [PSL9 IP](#psl9-ip) for more information on PSL IP generation).
+The build process for the CAPI 2.0 board support depends on POWER9 PSL (PSL9) sources that can be obtained as zip archive from the IBM Portal for OpenPOWER at
+
+https://www.ibm.com/systems/power/openpower
```

Comment:
Can we put the direct link to the zip file (as hidden behind the visible link [https://www.ibm.com/systems/power/openpower](https://www-355.ibm.com/systems/power/openpower/posting.xhtml?postingId=1BED44BCA884D845852582B70076A89A) link?
This would ensure user arrives directly to the right zip file.